cookiesales.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+//Select the HTML elements we will be modifing
+var displayResult = document.getElementById('tableResults');
+var addNewStore = document.getElementById('newStoreAdd');
+
+//Add an Event Lisnter
+addNewStore.addEventListener('submit', createNewStore);
+
+//Create the listner handler function
+function createNewStore(event) {
+  event.preventDefault();
+  var getTarget = event.target;
+
+//get Values from form inputs
+  var newStoreNameAdd = getTarget.newStoreNameAdd.value;
+  var newStoreMinCustomer = getTarget.newStoreMinCustomer.value;
+  var newStoreMaxCustomer = getTarget.newStoreMaxCustomer.value;
+  var newStoreAverCustomer = getTarget.newStoreAverCustomer.value;
+
+  getTarget.newStoreNameAdd.value = '';
+  getTarget.newStoreMinCustomer.value = '';
+  getTarget.newStoreMaxCustomer.value = '';
+  getTarget.newStoreAverCustomer.value = '';
```

Comment:
Look into HTML forms on MDN.  There is a reset method you have available that can reduce this to a single line.